### PR TITLE
implement create personal access token

### LINKED ISSUE #1087: Implement Create a personal access token
It would be nice to have this feature implemented, as I cannot find it in this library yet.

GitLab Docs: https://docs.gitlab.com/ee/api/users.html#create-a-personal-access-token

I could try to submit a PR if no one else comes up

### PR DESCRIPTION
**Pull request by FalcoSuessgott**
Fixes #1087

### CODE REVIEW

**Comment by FalcoSuessgott** [comment]
@svanharmelen 
Is there a problem with this PR or should I adjust something?
Kind regards

File: users.go [hunk 1]
```diff
@@ -828,6 +828,54 @@ func (s *UsersService) RevokeImpersonationToken(user, token int, options ...Requ
...
+}
+
+// CreatePersonalAccessTokenOptions represents the available
+// CreatePersonalAccessToken() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/users.html#create-a-personal-access-token
+type CreatePersonalAccessTokenOptions struct {
+	Name      *string    `url:"name,omitempty" json:"name,omitempty"`
+	ExpiresAt *time.Time `url:"expires_at,omitempty" json:"expires_at,omitempty"`
```
Comment (svanharmelen) [inline users.go:854]:
```suggestion
	ExpiresAt *ISOTime  `url:"expires_at,omitempty" json:"expires_at,omitempty"`
```

**Review by svanharmelen — COMMENTED** [review]
One left... Right?

**Comment by FalcoSuessgott** [comment]
good catch :) 
Hope now everything looks good.


BTW While implementing the changes I noticed that there might are some missing attributes (for e.g `userID`) in the `ImpersonationToken` struct.. Maybe u can verify it.

**Review by svanharmelen — APPROVED** [review]
Thanks @FalcoSuessgott! LGTM!

**Pull request merged**